src/resources/ClusterEvents/ClusterEventDetails.js

```diff
@@ -0,0 +1,16 @@
+import { useParams } from 'react-router-dom';
+import EventDetails from 'resources/Events/EventDetails';
+
+export const ClusterEventDetails = props => {
```

Comment:
Could you not use `props`, and add proper properties that we need? `resourceName` etc